lib/matplotlib/table.py

```diff
@@ -277,11 +278,32 @@ class Table(Artist):
              'top':          16,
              'bottom':       17,
              }
+    """Possible values where to place the table relative to the Axes."""
 
     FONTSIZE = 10
-    AXESPAD = 0.02    # the border between the axes and table edge
+
+    AXESPAD = 0.02
+    """The border between the Axes and the table edge."""
 
     def __init__(self, ax, loc=None, bbox=None, **kwargs):
+        """
+        Parameters
+        ----------
+        ax : `matplotlib.axes.Axes`
+            The `~axes.Axes` to plot the table into.
+        loc : str
+            The position of the cell with respect to *ax*. This must be one of
+            the `~.Table.codes`.
+        bbox : `.Bbox` or None
+            A bounding box to draw the table into. If this is not *None*, this
+            overrides *loc*.
+
+        Other Parameters
+        ----------------
+        **kwargs
+            `.Artist` properties.
+
```

Comment:
Minor preference for no empty line at end of docstring (that makes *three* lines with effectively no content (empty, closing quotes, empty)).